tests/php/core-api/wpcom-fields/test-post-fields-publicize-connections.php

```diff
@@ -1,7 +1,6 @@
 <?php
 
-require_once JETPACK__PLUGIN_DIR . '/tests/php/lib/class-wp-test-jetpack-rest-testcase.php';
-require_once JETPACK__PLUGIN_DIR . '/tests/php/lib/class-wp-test-spy-rest-server.php';
+require_once dirname( dirname( __DIR__ ) ) . '/lib/class-wp-test-jetpack-rest-testcase.php';
```

Comment:
What about keeping the `JETPACK__PLUGIN_DIR` constant?

Reply:
`JETPACK__PLUGIN_DIR` is not defined in WPCOM context. So to avoid conditional environment check we need to rely on relative paths.